lib/akamai_rspec/request.rb

```diff
@@ -75,6 +75,10 @@ def get(url, headers = {})
         return AkamaiRSpec::Response.new(url)
       end
 
+      if url.is_a? AkamaiRSpec::Response
+        return url
+      end
```

Comment:
The allowed response argument type is `RestClient::Response`, and there is a check for that case on lines 73-76. This if statement is not necessary.